pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -1038,7 +1038,7 @@ def process_circuits(
                                 "max_classical_register_width", 32
```

Comment:
If we want to change the default we should change it here too, but I'm not sure if we want to do this.